src/utils.js

```diff
@@ -1201,3 +1201,13 @@ export function mergeDeep(target, ...sources) {
 
   return mergeDeep(target, ...sources);
 }
+
+export function _getPageUrl(bidRequest, bidderRequest) {
```

Comment:
I know we've already updated many adapters but, in order to modify as less library files as possible, I'd move this to [marfeelTools.js](https://github.com/Marfeel/Prebid.js/blob/master/src/marfeelTools.js). What do you think?

Reply:
Fixed